kubernetes/resource_kubernetes_kubesystem_namespace.go

```diff
@@ -0,0 +1,52 @@
+package kubernetes
+
+import (
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func resourceKubernetesKubeSystemNamespace() *schema.Resource {
+	// reuse kubernetes_namespace schema, and methods for READ, UPDATE
+	kubeSystemNamespace := resourceKubernetesNamespace()
+	kubeSystemNamespace.Create = resourceKubernetesKubeSystemNamespaceCreate
+	kubeSystemNamespace.Delete = resourceKubernetesKubeSystemNamespaceDelete
+
+	// cidr_block is a computed value for Default VPCs
+	kubeSystemNamespace.Schema["cidr_block"] = &schema.Schema{
```

Comment:
This field from `aws_default_vpc` should certainly be dropped.

Reply:
Oops - nice catch